__test__/url-helper.test.ts

```diff
@@ -0,0 +1,33 @@
+import * as urlHelper from '../src/url-helper'
+
+describe('isGhes tests', () => {
+  it('basics', async () => {
+    expect(urlHelper.isGhes()).toBeFalsy()
+    expect(urlHelper.isGhes('https://github.com')).toBeFalsy()
+    //expect(urlHelper.isGhes('https://api.github.com')).toBeFalsy()
+    expect(urlHelper.isGhes('https://europe.ghe.com')).toBeFalsy()
```

Comment:
Swapping with something that more closely resembles a tenant.

```suggestion
    expect(urlHelper.isGhes('https://contoso.ghe.com')).toBeFalsy()
```